### PR TITLE
Cleanup adapter after processing styles

### PR DESCRIPTION
**Pull request by mattcompiles**

### CODE REVIEW

File: .github/workflows/validate.yml [hunk 1]
```diff
@@ -28,3 +28,34 @@ jobs:
 
       - name: Test
         run: yarn test:jest
+  playwright:
```
Comment (mattcompiles) [inline .github/workflows/validate.yml:31]: Unrelated cleanup

File: packages/integration/package.json [hunk 2]
```diff
@@ -3,7 +3,6 @@
   "version": "1.2.0",
   "description": "Zero-runtime Stylesheets-in-TypeScript",
   "main": "dist/vanilla-extract-integration.cjs.js",
-  "module": "dist/vanilla-extract-integration.esm.js",
```
Comment (mattcompiles) [inline packages/integration/package.json:6]: Decided to only export a CJS version of the integration as there was no reason to have ESM

**Review by markdalgleish — APPROVED** [review]
